lib/parse_message.js

```diff
@@ -13,6 +13,21 @@ var replyFor = require('./codes');
 module.exports = function parseMessage(line, stripColors) {
     var message = {};
     var match;
+    
+    // twitch.tv-specific, twitch stuffs a bunch of useful information here
+    // need to do a client.send("CAP REQ", "twitch.tv/tags") first
+    if(line.charAt(0) == "@") {
+        var tagend = line.indexOf(" ");
```

Comment:
variables declarations for aliases should not be used, but if you REALLY NEED them aliases, you should declare the variable at function-start and not on a if-statement.